modules/ROOT/pages/_partials/sync/creating-a-voyager-server-dockerfile-for-use-with-openshift.adoc

```diff
@@ -0,0 +1,59 @@
+[id="creating-a-voyager-server-dockerfile-for-use-with-openshift-{context}"]
+= Creating a voyager server container image for use with OpenShift
+
+This section describes how to create a container image using a local voyager server repo.
+
+Although it is not covered in this section, you can add the Apollo Voyager Client.
+For more information, see xref:server-getting-started-with-an-example-data-source-{context}[Getting started with an example data source].
+
+.Prerequisites
+
+* You have cloned the link:https://github.com/aerogear/voyager-server[Apollo Voyager Server] locally.
+* Docker is installed and is running on your local machine. Use `sudo systemctl start docker` to start docker. As an alternative, use `podman`, or `buildah`.
+
+.Procedure
+
+. Open a terminal and browse to the Apollo Voyager Server repo on your local machine.
++
+. Create a Dockerfile, for example, `touch Dockerfile`.
++
+. Open the Docker file and add the following content, then save the changes:
```

Comment:
suggestion:

combine 1, 2 and 3 into:

. Create a file named `Dockerfile` in the root directory of the Apollo Voyager Server repo, with the following contents: